# HG changeset patch
# User Wolfgang Rosenauer <wr@rosenauer.org>
# Date 1558442924 -7200
#      Tue May 21 14:48:44 2019 +0200
# Node ID beca42e6765b14ff0f69bb056678b65fedd7a39e
# Parent  a9195ad0464a3d1797630bca010dfbb53fe12027
imported patch mozilla-openaes-decl.patch

diff -r a9195ad0464a -r beca42e6765b media/gmp-clearkey/0.1/openaes/oaes_lib.c
--- a/media/gmp-clearkey/0.1/openaes/oaes_lib.c	Tue May 21 14:48:40 2019 +0200
+++ b/media/gmp-clearkey/0.1/openaes/oaes_lib.c	Tue May 21 14:48:44 2019 +0200
@@ -29,6 +29,7 @@
  */
 
 #include <stdlib.h>
+#include <stdio.h>
 #include <stddef.h>
 #include <time.h> 
 #include <string.h>
